src/components/Contacts/ContactDetail/ContactDetailTab/ContactDetailTab.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  Box,
+  Button,
+  makeStyles,
+  Theme,
+  ThemeProvider,
+} from '@material-ui/core';
+import { Create } from '@material-ui/icons';
+import React from 'react';
+import theme from '../../../../theme';
+import { ContactDetailQuery } from '../ContactDetail.generated';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  contactDetailHeadingContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignContent: 'center',
+    alignItems: 'center',
+  },
+  contactDetailHeadingText: {
+    flexGrow: 5,
+  },
+  contactDetailHeadingIcon: {
+    width: '18px',
+    height: '18px',
+    margin: theme.spacing(0),
+    color: '#9C9FA1',
+  },
+  contactDetailContainer: {
+    padding: theme.spacing(1),
+  },
+}));
+interface ContactDetailTabProps {
+  contact: ContactDetailQuery;
+}
+
+export const ContactDetailTab: React.FC<ContactDetailTabProps> = ({
+  contact,
+}) => {
+  const classes = useStyles();
+  const calledContact = contact.contact;
+
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
This is provided globally for the app and Storybook. At this point tests may still need to include it.
https://github.com/CruGlobal/mpdx-react/blob/main/pages/_app.page.tsx#L85
https://github.com/CruGlobal/mpdx-react/blob/main/.storybook/preview.js#L31

Reply:
Thanks, I forgot to remove this.